doc/README.md

```diff
@@ -32,8 +32,8 @@ reference manual requires Python 3, and the following Python packages:
 
   - sphinx >= 3.0.2
   - sphinx_rtd_theme >= 0.4.3
-  - beautifulsoup4 >= 4.0.6
-  - antlr4-python3-runtime >= 4.7.1
+  - beautifulsoup4 >= 4.6.0
```

Comment:
I actually ended up using 4.0.5 in my local tests for #17772, although the exact version of `beautifulsoup4` is not specified in the dockerfile.

Reply:
(For future reference, the reason this needs changing is that `beautifulsoup4` does not have a version 4.0.6, so the previous documentation was probably a typo.)